docs/UserGuide.md

```diff
@@ -18,26 +18,27 @@ VolunSync is a **desktop app for keeping track of volunteers and volunteering ev
 
 1. Copy the file to the folder you want to use as the _home folder_ for your VolunSync application.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar volunsync.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Open your computer's command terminal:
+    - On Windows: Press `Windows + R`, type `cmd`, and hit Enter.
+    - On Mac/Linux: Open the Terminal from your Applications.
+1. In the terminal, type `cd` followed by the path to the folder with the .jar file, then run the following command:
+   java -jar volunsync.jar
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the User Guide on your browser.<br>
    Some example commands you can try:
 
-   * `reset` : Lists all volunteers and events in the VolunSync database.
+    * `list` : Lists all volunteers and events in the VolunSync database.
 
-   * `/v new n/ John Doe p/ 91234567 em/ john@gmail.com d/ 2024-02-02 s/ 00:10 e/ 23:59` : Creates a new record for a volunteer named `John Doe` to the VolunSync database.
+    * `/v new n/ John Doe p/ 91234567 em/ john@gmail.com d/ 2024-02-02 s/ 00:10 e/ 23:59` : Creates a new record for a volunteer named `John Doe` to the VolunSync database.
 
-   * `/v del 3` : Deletes the third volunteer on the list of volunteers.
+    * `/v del 3` : Deletes the third volunteer on the list of volunteers.
 
-   * `/e new n/ Coding Exam s/ 19:00 e/ 21:00 d/ 2024-12-12 l/ LT 28` : Creates a new record for an event named `Coding Exam` to the VolunSync database.
+    * `/e new n/ Flag Day s/ 19:00 e/ 21:00 d/ 2024-12-12 l/ LT 28` : Creates a new record for an event named `Flag Day` to the VolunSync database.
```

Comment:
Even though the order of parameters does not matter, we should make it consistent with our warning msg e to user:
"new: Adds an event to the calander. Parameters: n/NAME l/LOCATION d/DATE s/START TIME e/END TIME des/DESCRIPTION Example: /e new n/Food collection l/NTUC d/2024-11-29 s/00:00 e/23:59 des/Collecting unsold food from NTUC for distribution"
So name, location, date, start time, and end time and optionally description.

Reply:
Updated Example